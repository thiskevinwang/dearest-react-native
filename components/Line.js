// @flow
import * as React from "react";
import { View, StyleSheet } from "react-native";

type Props = {};

export default function Line(props: Props) {
  return <View style={[styles.line, props.style]} />;
}

const styles = StyleSheet.create({
  line: {
    borderBottomColor: "#eee",
    borderBottomWidth: 1
  }
});
