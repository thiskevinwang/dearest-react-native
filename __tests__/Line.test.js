import "react-native";
import React from "react";
import Line from "../components/Line";
import renderer from "react-test-renderer";

describe("<Line />", () => {
  it("should render correctly", () => {
    const line = renderer.create(<Line />).toJSON();
    expect(line).toMatchSnapshot();
  });
});
