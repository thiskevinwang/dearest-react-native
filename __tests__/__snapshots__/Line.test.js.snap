// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Line /> should render correctly 1`] = `
<View
  style={
    Array [
      Object {
        "borderBottomColor": "#eee",
        "borderBottomWidth": 1,
      },
      undefined,
    ]
  }
/>
`;
